Ecosystem Partners/Marketplace Guides/getting-started-with-avni-software-defined-cloud-partner-template.md

```diff
@@ -0,0 +1,153 @@
+{{{
+"title": "Getting Started with Avni Software Defined Cloud - Partner Template",
+"date": "8-13-2015",
+"author": "Bob Stolzberg",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+![logo](http://www.avni.io/media/photologue/photos/cache/Avni-Logo-Tagline-final-20150408_G7m1fR2_730.png)
```

Comment:
I'm skeptical to link off to third party for images... this is called hot linking and is generally frowned upon. Can you down the logo and include it instead?